src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -306,12 +306,15 @@ const buildWhereClause = ({ schema, query, index }): WhereClause => {
       patterns.push(`$${index}:name = $${index + 1}`);
       values.push(fieldName, fieldValue);
       index += 2;
-    } else if (fieldName === '$or' || fieldName === '$and') {
+    } else if (['$or', '$nor', '$and'].includes(fieldName)) {
       const clauses = [];
       const clauseValues = [];
       fieldValue.forEach((subQuery) =>  {
         const clause = buildWhereClause({ schema, query: subQuery, index });
         if (clause.pattern.length > 0) {
+          if (fieldName === '$nor') {
+            clause.pattern = `(NOT ${clause.pattern})`;
+          }
```

Comment:
you're doing `(NOT query1) AND (NOT query2)` instead of `NOT (query1 OR query2)` whhich tecnically work but can be a bit misleading for maintenance. Cna you document it?

Reply:
True I was pretty lazy 😅

Reply:
Yeah I figured it worked but letMs write it cleanly please

Reply:
I'll fix that